src/System.Threading.AccessControl/src/System/Security/AccessControl/EventWaitHandleSecurity.cs

```diff
@@ -117,20 +117,20 @@ public EventWaitHandleSecurity()
 
         [System.Security.SecurityCritical]  // auto-generated
```

Comment:
Unrelated to the feedback on the previous PR, but can these attributes be removed?